src/lib/components/commons/FilterPanel/FilterPanel.styled.ts

```diff
@@ -36,11 +36,11 @@ export const StyledFilterButton = styled(StyledButton)<IStyledFilterProps>`
     isButtonGroup &&
     css`
       border-radius: 0;
-      &:last-of-type {
+      &:last-of-type I{
```

Comment:
what is that?

Reply:
lol its beacause of the change-prefix script!
it iditified 'type' and add capital I after...